setup.py

```diff
@@ -42,27 +45,28 @@ def read(*rnames):
           'License :: OSI Approved :: Zope Public License',
           'Programming Language :: Python',
           'Programming Language :: Python :: 3',
-          'Programming Language :: Python :: 3.6',
           'Programming Language :: Python :: 3.7',
           'Programming Language :: Python :: 3.8',
           'Programming Language :: Python :: 3.9',
           'Programming Language :: Python :: 3.10',
+          'Programming Language :: Python :: 3.11',
           'Programming Language :: Python :: Implementation :: CPython',
           'Natural Language :: English',
           'Operating System :: OS Independent',
           'Topic :: Internet :: WWW/HTTP',
-          'Framework :: Zope :: 3'],
+          'Framework :: Zope :: 3',
+      ],
       keywords="zope3 grok grokadmin",
       author="Zope Foundation and Contributors",
-      author_email="grok-dev@zope.org",
+      author_email="grok-dev@zope.dev",
```

Comment:
There's no grok-dev email list, either. Does anyone actually use the grok packages?

```suggestion
      author_email="zope-dev@zope.dev",
```